### DISCUSSION TITLE
Directive i can't understand in the code of v-table component

### DISCUSSION THREAD

**Discussion by massimans** [directus/directus#2844] [Archive]
Hi,
When wondering in the code base i found this strang bit i can't understand in the v-table component
#[`header.${header.value}`]
Can Any one please help me with that

**Comment by rijkvanzanten** [comment]
It's dynamically selecting the correct slot for the header, so 

```
#[`header.${header.value}`]
```

will parse to

```
#header.title
```

which is a shorthand for

```
v-slot:header.title
```

**Comment by massimans** [comment]
Thanks a lot i missed the new v-slot syntax ,i was hoping this was how the the tabular layout selected the correct interface for the fields, can you help me with that, I need it to create new layouts I need
